parsers/ENTSOE.py

```diff
@@ -623,8 +642,11 @@ def query_generation_forecast(
 
     # Note: this does not give a breakdown of the production
     params = {
-        "documentType": "A71",  # Generation Forecast
-        "processType": "A01",  # Realised
```

Comment:
Incorrect comment here about Realised btw.

Reply:
I think this was just copied from another process and then modified and the comment slipped by, maybe by me 😅